src/server/InputHandler.ts

```
@@ -46,20 +43,21 @@ export class InputHandler {
                 break;
 
             case 'scroll':
-                const invertMultiplier = (CONFIG.MOUSE_INVERT ?? false) ? -1 : 1;
                 const promises: Promise<void>[] = [];
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)